### PR TITLE
1009/leanpub

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1009 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,35 +65,26 @@ You can also read the MSTG on [Gitbook](https://mobile-security.gitbook.io/mobil
...
 
 2. Check the [releases](https://github.com/OWASP/owasp-mstg/releases "Our releases"). Here you can find a PDF, an archive containing the sources and a DocX document for any given tagged version. Please note that the documents are generated automatically per tag.
 
 3. Get the [e-book](https://leanpub.com/mobile-security-testing-guide-preview "MSTG as an e-book"). The book is available for free, but you can choose to purchase it at a price of your choosing if you wish to support our project. All funds raised through sales of the e-book go directly into the project budget and will be used to fund production of future releases.
 
-
-3. Clone the repository and run the [document generator](https://github.com/OWASP/owasp-mstg/blob/master/Tools/generate_document.sh "The document generator") (requires [pandoc](http://pandoc.org "Pandoc")). This produces docx and html files in the "Generated" subdirectory.
+3. Clone the repository and run the [document generator](https://github.com/OWASP/owasp-mstg/blob/master/Tools/generate_document.sh "The document generator") (requires [pandoc](http://pandoc.org "Pandoc")). This produces docx and HTM files in the "Generated" subdirectory.
```
Comment (TheDauntless) [inline README.md]: Can't check due to bad internet, but I assume it generates HTML files (<-> HTM)

File: Tools/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Tools
+
+## Overview
+
+This directory is for tools that are used to generate the necessary files for our release-channels.
+
+Channels:
+- Gitbook: currenlty using @sushi2k his repository who is synced manually.
```
Comment (TheDauntless) [inline Tools/README.md]: > Gitbook: currently using @sushi2k's repository which is synced manually.

File: Tools/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- Github release: currently using travis to automatically build stuff based on a tag. See ../travis.yml. It uses `gendocs.sh`, `generate_document.sh` and `gitbookepubanpdf.sh`.
+- Leanpub: `updateLeanpub.sh` is in the making: for now it hold instructions only.
```
Comment (TheDauntless) [inline Tools/README.md]: > - Leanpub: `updateLeanpub.sh` is in the making: for now it contains only instructions.

File: Tools/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Files:
+- book.json : the book.json template for gitbook. Necessary for `gitbookepubanpdf.sh` to automatically create an updated book.json in the root of the folder.
```
Comment (TheDauntless) [inline Tools/README.md]: > - book.json: the book.json template for gitbook. Necessary for `gitbookepubanpdf.sh` to automatically create an updated book.json in the root of the folder.

Comment (TheDauntless) [reply Tools/README.md]: (space before : )

File: Tools/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- gendocs.sh: used to simplify the work with Travis.
+- generate_document.sh: used to generate the docx and html files.
+- generate_toc.rb: used to generate a TOC file
```
Comment (TheDauntless) [inline Tools/README.md]: > - generate_toc.rb: used to generate a TOC file.

File: Tools/README.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- gitbookepubandpdf.sh: used to generate the epub, pdf and mobi files.
+- metadata.yml: used by pandoc for generating docx files with `generate_Document.sh`
```
Comment (TheDauntless) [inline Tools/README.md]: > - metadata.yml: used by pandoc for generating docx files with `generate_Document.sh`.


(period)

File: Tools/README.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+- omtg-pre-commit.sh: older not maintained hook for automatically triggering generate_toc.rb
```
Comment (TheDauntless) [inline Tools/README.md]: > - omtg-pre-commit.sh: older not maintained hook for automatically triggering generate_toc.rb.

(period)

File: Tools/README.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+- reference.docx : templatefile used for generating the word doc using `generate_document.sh`.
```
Comment (TheDauntless) [inline Tools/README.md]: > - reference.docx : templatefile used for generating the word doc using `generate_document.sh`.

(space before : )

File: Tools/README.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+## Release process:
+1. Sync @sushi2k his repository
```
Comment (TheDauntless) [inline Tools/README.md]: > Sync @sushi2k's repository

File: Tools/README.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+2. Update the Changelog.md
+3. Commit the changes (with message "Release <version>")
+4. Push a tag with the new version (git tag -a <version> -m "Release message that will be on github")
+5. Update the Leanpub Files
+6. Update Owasp Wiki if necessary
```
Comment (TheDauntless) [inline Tools/README.md]: OWASP

File: README.md (hunk 1 above)
Comment (commjoen) [reply README.md]: 👍

File: Tools/README.md (hunk 3 above)
Comment (commjoen) [reply Tools/README.md]: 👍

File: Tools/README.md (hunk 4 above)
Comment (commjoen) [reply Tools/README.md]: 👍

**Pull request merged**